tenderly/unchainedProxy.ts

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable @shapeshiftoss/logger/no-native-console */
+import type { evm } from '@shapeshiftoss/common-api'
+import dotenv from 'dotenv'
+import express from 'express'
+import { readFileSync } from 'fs'
+import { createProxyMiddleware, fixRequestBody } from 'http-proxy-middleware'
+import Web3 from 'web3'
+import type { Transaction } from 'web3-types'
+import { hexToNumber, numberToHex } from 'web3-utils'
+import type { RawData, WebSocket as WebSocketClient } from 'ws'
+import WebSocket, { WebSocketServer } from 'ws'
+
+import { corsMiddleware } from './middleware/cors'
+import {
+  PROXY_UNCHAINED_ETHEREUM_HTTP_PORT,
+  PROXY_UNCHAINED_ETHEREUM_WS_PORT,
+} from './utils/constants'
+import { getTenderlyRpcUrl } from './utils/getTenderlyRpcUrl'
+import { estimateGas, getBalance, getTransactionByHash, sendTransaction } from './utils/rpcHelpers'
+
+const { REACT_APP_UNCHAINED_ETHEREUM_HTTP_URL, REACT_APP_UNCHAINED_ETHEREUM_WS_URL } = dotenv.parse(
+  readFileSync('.env.dev'),
+)
+
+const httpPort = Number(PROXY_UNCHAINED_ETHEREUM_HTTP_PORT)
+const websocketPort = Number(PROXY_UNCHAINED_ETHEREUM_WS_PORT)
+const provider = new Web3.providers.HttpProvider(getTenderlyRpcUrl())
```

Comment:
Could use `JsonRpcBatchProvider` for consistency - though that would imply using ethers vs. web3 and would be a huge lift, so obviously disregard this, just something to keep in mind if we want to improve performance in the future. ethers JSON-RPC providers also support [jsonRpcProvider.send](https://docs.ethers.org/v5/api/providers/jsonrpc-provider/#JsonRpcProvider-send)